views/NoResults/index.jsx

```diff
@@ -6,6 +6,9 @@ const NoResults = (props) => (
     <p className="base--h2">
       There are no results for: {props.query ? props.query.text : '' }
     </p>
+    <p className="base--h3" style={props.query.restrictedDateRange ? {display: 'block'} : {display: 'none'}}>
```

Comment:
Also, this is mostly just a style comment,  I would change line 9 to be something more like:

`<p className={props.restrictedDateRange ? 'base--h3' : 'base--h3 hidden'}>`

instead of embedded `style`.